spectacles.gemspec

```diff
@@ -22,8 +22,8 @@ Gem::Specification.new do |gem|
   # Dependencies
   #
   gem.required_ruby_version = ">= 2.0.0"
-  gem.add_dependency "activerecord", ">= 3.2.0"
-  gem.add_dependency "activesupport", ">= 3.2.0"
+  gem.add_dependency "activerecord", ">= 3.2.0", "< 6.0"
```

Comment:
Haven't tested anything in Rails 6, so I thought this was prudent.

Reply:
That's reasonable. It also allows us to easily support it moving forward.